blueprints/addon/files/.travis.yml

```diff
@@ -24,7 +28,17 @@ matrix:
   allow_failures:
     - env: EMBER_TRY_SCENARIO=ember-canary
 
-before_install:
+<% if (yarn) { %>before_install:
+  - curl -o- -L https://yarnpkg.com/install.sh | bash
+  - export PATH=$HOME/.yarn/bin:$PATH
+  - yarn global add bower phantomjs-prebuilt
+  - bower --version
+  - phantomjs --version
+
+install:
+  - yarn install
```

Comment:
@Turbo87 - Why? To force fresh deps? If that is the case, then `ember-try` scenarios will already test against latest.

I'm not opposed to changing to `--no-lockfile` though